app/models/journeys/early_years_payment/provider/slug_sequence.rb

```diff
@@ -0,0 +1,29 @@
+module Journeys
+  module EarlyYearsPayment
+    module Provider
+      class SlugSequence
+        SLUGS = %w[].freeze
+
+        def self.start_page_url
+          if Rails.env.production?
+            "https://www.example.com" # TODO: update to correct guidance
```

Comment:
For FE i ended up removing this. from what I could tell, stakeholders wanted the landing page to be the start page

Reply:
ahh thank you! i think that's the same for EY - i'll remove too 👍